soccer/gameplay/tactics/positions/goalie.py

```diff
@@ -195,6 +195,11 @@ def execute_block(self):
 
     def execute_defend(self):
         dest_x = main.ball().pos.x / constants.Field.Width * Goalie.MaxX
+        if main.ball().pos.y <= constants.Field.Width / 8:
```

Comment:
This file's changes can be removed. You can do this with:

`git revert 7bdec2b`